packaging/test_irods_client_rest_cpp.py

```diff
@@ -70,6 +70,65 @@ def test_access_with_default_arguments(self):
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
 
+    @unittest.skip('Not relevant on this branch...')
+    def test_logical_path_rename(self):
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                coll = admin.home_collection + "/coll"
+
+                # test on a data object
+                new_coll = coll.replace('coll', 'new_coll')
+                dobj = new_coll + "/data_object"
+                new_dobj = dobj.replace("data_object", "new_data_object")
+                admin.assert_icommand(['imkdir', coll])
+                res = irods_rest.logical_path_rename(
+                    token,
+                    coll,
+                    new_coll
+                )
+                self.assertEqual(res, "")
+                # test on a collection
+                # make sure that it works with a collection that's actually populated
+                admin.assert_icommand(['itouch', dobj])
+                res = irods_rest.logical_path_rename(
+                    token,
+                    dobj,
+                    new_dobj
+                )
+                self.assertEqual(res, "")
+            finally:
+                admin.run_icommand(['irm', '-r', '-f', new_coll])
+
+
+    @unittest.skip('Not relevant on this branch...')
+    def test_logical_path_delete(self):
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+        with session.make_session_for_existing_admin() as admin:
+            # test on data object
+            coll_path = admin.home_collection + '/coll'
+            dobj_path = coll_path + '/data_object'
```

Comment:
let's get more descriptive names for coll and data_object... consider putting issue numbers in the test file names...